documentation/modules/exploit/multi/php/car_rental_system.md

```diff
@@ -0,0 +1,76 @@
+## Vulnerable Application
+
+Car Rental System 1.0 is a Web Application to rent cars. It was write using PHP and MySQL.
+
+Vendor Homepage: https://www.sourcecodester.com/php/14544/car-rental-management-system-using-phpmysqli-source-code.html
+Software Link: https://www.sourcecodester.com/download-code?nid=14544&title=Car+Rental+Management+System+using+PHP%2FMySQLi+with+Source+Code
+
+to install, extract all files in a web directory, edit admin/db_connect.php, and create database following database/car_rental_db.sql
+
+Opening the web page, we can login in /admin area using " ' OR 1=1 limit 1# " as username, and a random word as password; Giving "limit 1" we will select the first user: ADMIN
+
+This software is also vulnerable for Arbitrary File Upload.
+
+Vulnerable page: http://IP/Car_CMSHOME/admin/index.php?page=manage_car. Editing a Car record, you may upload a php file.
```

Comment:
This doesn't really describe things properly. Let me show you how I would explain this a bit better:

```suggestion
Once authenticated as an administrator, attackers can then upload arbitrary files via the `/admin/ajax.php?action=save_car` page, which will then be placed under the `/admin/` directory on the web server. By uploading
an executing a malicious PHP file, attackers can gain arbitrary code execution on the web server as the `www-data` user.
```